.changeset/gold-beds-compare.md

```diff
@@ -0,0 +1,5 @@
+---
+"chainlink": patch
+---
+
+Modified FeeQuoter's TokenPriceFeedConfig struct to support tokens with zero decimals #bugfix
```

Comment:
I think we're good without `core` changesets when the only change to core is a gethwrapper update

Reply:
I agree but the CI wouldn't stop complaining about it not existing.